frontend/src/components/Pages/Dashboard/EvaluationForm.jsx

```
@@ -0,0 +1,417 @@
+          <span className="text-xs text-red-500 font-medium">Required</span>
+        )}
+      </div>
+      
+      <div className="flex items-center justify-between gap-1">
+        {ratings.map((rating) => (
+          <div key={rating} className="flex flex-col items-center">
+            <Radio
+              id={`${question.id}-${rating}`}
+              name={`rating-${question.id}`}
+              value={rating}
+              checked={parseInt(value) === rating}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)